static/styles/portico.css

```diff
@@ -1696,7 +1696,7 @@ input.new-organization-button {
     margin-top: -240px;
     height: auto;
     width: calc(100% - 40px);
-    max-width: calc(768px + 20px);
+    max-width: calc(768px - -20px);
```

Comment:
I couldn't find where this one is being used even. I checked out almost all portico related pages but never this got rendered.
@brockwhittaker where is this used?

Reply:
I think it's in the footer of the /help/ pages.

Reply:
It's possible we got rid of this footer in the HTML.